dashboard/src/components/branding/che-dashboard-configuration.service.ts

```diff
@@ -48,4 +57,45 @@ export class CheDashboardConfigurationService {
     });
   }
 
+  enabledFeature(feature: che.TogglableFeature): boolean {
+    const disabledFeatures = this.cheBranding.getConfiguration().features.disabled;
+    return disabledFeatures.indexOf(feature) === -1;
+  }
+
+  getFooterLinks(): { [key: string]: FooterLink } {
```

Comment:
I think we should use the Map. WDYT?
```
getFooterLinks(): Map<string, FooterLink> {
...
```

Reply:
I don't see any benefits of using the Map. Can you elaborate on this please?

Reply:
As for me it is simpler to use map.set(), map.get(), map.has() and map.size 

A minor one. Just a proposal.

Reply:
Using Map in this case is redundant as to me. Let's leave it as is.